newrelic/plugin.spec.yaml

```diff
@@ -49,7 +49,7 @@ types:
       type: string
 actions:
   list_users:
-    title: List User
+    title: List Users
```

Comment:
Major version bump

Reply:
I reverted back, I just want to avoid version bumps unless I have to.